packages/client/src/orders/types/orderItem.types.ts

```diff
@@ -87,6 +87,8 @@ export type OrderItem = {
   productType: keyof typeof ProductType;
   selectedSaleIntent?: SelectedSaleIntent | string;
   metadata?: Record<string, string>;
+  expectedFulfillmentDate?: string;
+  saleIntent?: string;
```

Comment:
What is the difference between `saleIntent` and `selectedSaleIntent` that is also defined in this `OrderItem` type?

Reply:
As we talked in private, I've checked with the BE team and the `selectedSaleIntent` property does not exist. So it was removed to only keep `saleIntent`. I've also renamed the type since having the `selected` keyword no longer makes sense. Please validate if everything is ok 🙂